lstmcpipe/utils.py

```diff
@@ -80,7 +182,194 @@ def dump_lstchain_std_config(filename='lstchain_config.json', allsky=False, over
     print(f"Updating std lstchain config {extra_msg} with")
     diff = DeepDiff(std_cfg, cfg)
     pprint(diff)
-    
+
     with open(filename, 'w') as file:
         json.dump(cfg, file, indent=4)
     print(f"\nModified lstchain config dumped in {filename}. Check full config thoroughly.")
+
+
+def run_command(*args):
```

Comment:
why `rerun_command` with a max of `ntry=1` can't be used here?

Reply:
- `rerun_command` is using sp.run, while `run_command` sp.Popen 
 - rerun is expected for lstchain entry points while run for sbatch commands, see [here](https://stackoverflow.com/questions/39187886/what-is-the-difference-between-subprocess-popen-and-subprocess-run#:~:text=The%20main%20difference%20is%20that,receive%20data%20to%20your%20process.) 
 - rerun should be based on run, and not inversely

Reply:
> rerun should be based on run, and not inversely
Should we implement it ?

Reply:
Once I understand what `run_command` purpose is, we might.